README.md

```diff
@@ -0,0 +1,24 @@
+# Essential Studio for Blazor - Diagram Builder
+
+The Blazor diagram builder application is used to create diagrams such as FlowChart and MindMap diagrams. In diagram builder application have all supports of diagram,, such as Print, Export, and Drawing Tool support, are available in the Diagram Builder application. You can interact with the diagram or create your own using shapes.
```

Comment:
The Blazor diagram builder application is used to create diagrams such as FlowChart and MindMap diagrams. In the diagram builder application, all support diagrams, such as Print, Export, and Drawing Tool support, are available in the Diagram Builder application. You can interact with the diagram or create your own using shapes.